internal/services/gcp_reservation_service.go

```diff
@@ -93,14 +90,14 @@ func CreateGCPReservation(w http.ResponseWriter, r *http.Request) {
 	IBClient, ibErr := clients.GetImageBuilderClient(r.Context())
 	logger.Trace().Msg("Creating IB client")
 	if ibErr != nil {
-		renderError(w, r, payloads.NewClientInitializationError(r.Context(), "image builder client", ibErr))
+		renderError(w, r, payloads.NewClientInitializationError(r.Context(), "image builder", ibErr))
```

Comment:
ditto

Reply:
Okay, great! I was not aware of this. I can see your point now. I should change this. :)